sources/store/keyed_error_source.go

```diff
@@ -80,9 +81,12 @@ func (k *keyedErrorHealthCheckSource) HealthStatus(ctx context.Context) health.H
 			},
 		}
 	}
-	params := make(map[string]interface{}, len(k.keyedErrors))
+	params := map[string]interface{}{}
 	for key, err := range k.keyedErrors {
 		params[key] = err.Error()
+		for k, v := range sources.ErrorToUnderlyingSafeParamsMap(err) {
+			params[k] = v
```

Comment:
do we care about multiple errors having the same params? Should we nest them somehow for this case where we have multiple errors?

Reply:
Sure, done